indico/modules/events/registration/fields/simple.py

```diff
@@ -212,7 +211,7 @@ def unprocess_field_data(cls, versioned_data, unversioned_data):
 
     @property
     def filter_choices(self):
-        return OrderedDict(self.wtf_field_kwargs['choices'])
+        return self.wtf_field_kwargs['choices']
```

Comment:
This fixes the error I mentioned on IRC:

```suggestion
        return dict(self.wtf_field_kwargs['choices'])
```